tests/src/resultcollection.hh

```diff
@@ -43,6 +43,38 @@ inline auto linearStressResultsOfSquare = []<typename NOP, typename FE>(NOP& non
   return std::make_tuple(feRequirements, expectedStress, getVertexPositions(fe));
 };
 
+inline auto linearVonMisesResultsOfSquare = []<typename NOP, typename FE>(NOP& nonLinearOperator, FE& fe) {
+  constexpr int vertices   = 4;
+  constexpr int quantities = 1;
+
+  Eigen::Matrix<double, vertices, quantities> expectedStress;
+  expectedStress << 1953.44932249, 1182.27663689, 1182.27663689, 0;
```

Comment:
Eigen also support the construction as follows https://eigen.tuxfamily.org/index.php?title=3.4#New_Major_Features_in_Core  then the matrix structure is more clear and you can also make the object const.

Also below.

Reply:
This is not really needed now. Only If you want, And keep it in mind for later.